azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/AzureAppService.java

```diff
@@ -79,7 +82,7 @@ public IWebAppDeploymentSlot deploymentSlot(String resourceGroup, String appName
     }
 
     public IWebAppDeploymentSlot deploymentSlot(WebAppDeploymentSlotEntity deploymentSlot) {
-        return null;
+        return new WebAppDeploymentSlot(deploymentSlot, this);
     }
 
     public AzureResourceManager getAzureResourceManager() {
```

Comment:
Some comments about this class:

NO. 1: renaming : AzureAppService --> AzureAppServiceHelper/AzureAppServiceUtil? actually, i think this class is only a static util class.

![image](https://user-images.githubusercontent.com/19339116/106248231-aa0d4e00-624b-11eb-8a56-c2aee9142d36.png)

No. 2: As azureResourceManager property is only required by 2 method. how about to let is as a method parameter?
![image](https://user-images.githubusercontent.com/19339116/106248291-bee9e180-624b-11eb-8ecd-a01fccb31229.png)